### PR TITLE
Units additions

### PR DESCRIPTION
**Pull request by donovaly**
add the following units: 
- mmol
- ml
- mW
- (m) bar
- kJ
- kWh
- eV
- C
- T
- (p, n, u, m) F
- (n, u, m) H

### CODE REVIEW

File: src/App/FreeCADInit.py [hunk 1]
```diff
@@ -696,16 +701,35 @@ def report(self,msg,func,*args,**kargs):
...
+App.Units.MilliFarad    = App.Units.Quantity('mF')
+App.Units.Farad         = App.Units.Quantity('F')
+
+App.Units.NanoHenry     = App.Units.Quantity('nH')
+App.Units.MicroHenry    = App.Units.Quantity('uH')
+App.Units.MilliHenry    = App.Units.Quantity('mH')
+App.Units.Henry         = App.Units.Quantity('H')
 
 App.Units.Joule         = App.Units.Quantity('J')
+App.Units.KiloJoule     = App.Units.Quantity('kJ')
```
Comment (vocx-fc) [inline src/App/FreeCADInit.py]: Should we add millijoule as well? Maybe it's useful for some small scale analysis of integrated circuits or something like that. I'm not sure of the scale of the units but I think electronics are sometimes designed considering power consumption in milliwats, so millijoule would seem appropriate.

Comment (donovaly) [reply src/App/FreeCADInit.py]: > Should we add millijoule as well?

Yes. I have more additions in the pipeline but the patch is already very big. So let's add further units in a next step.

Comment (donovaly) [reply src/App/FreeCADInit.py]: > Should we add millijoule as well?

OK, I added this now and also kW

**Comment by wwmayer** [comment]
Merged.

**Pull request closed without merging**